bin/tx-prover/src/commands/mod.rs

```diff
@@ -0,0 +1,118 @@
+use clap::Parser;
+use init::Init;
+use proxy::StartProxy;
+use serde::{Deserialize, Serialize};
+use worker::StartWorker;
+
+pub mod init;
+pub mod proxy;
+pub mod worker;
+
+/// Configuration of the CLI
+///
+/// It is stored in a TOML file, which will be created by the `init` command.
+/// It allows manual modification of the configuration file.
+#[derive(Serialize, Deserialize)]
+pub struct CliConfig {
+    /// List of workers to start
+    pub workers: Vec<WorkerConfig>,
+    /// Proxy configuration
+    pub proxy: ProxyConfig,
+}
+
+impl Default for CliConfig {
+    fn default() -> Self {
+        Self {
+            workers: vec![WorkerConfig::default(), WorkerConfig::default()],
+            proxy: ProxyConfig::default(),
+        }
+    }
+}
+
+/// Configuration for a worker
+#[derive(Serialize, Deserialize)]
+pub struct WorkerConfig {
+    pub host: String,
+    pub port: u16,
+}
+
+impl Default for WorkerConfig {
+    fn default() -> Self {
+        Self { host: "0.0.0.0".into(), port: 8080 }
```

Comment:
Should we use a different port? Port 8080 is [usually used for web traffic](https://www.speedguide.net/port.php?port=8080)